src/pages/MainPage.tsx

```diff
@@ -170,227 +174,324 @@ export const MainPage: React.FC<{}> = (props) => {
+
+							const formattedArray = await insert13Before10(
+								Uint8Array.from(Buffer.from(emailFull))
+							);
+							// Due to a quirk in carriage return parsing in JS, we need to manually edit carriage returns to match DKIM parsing
+							console.log('formattedArray', formattedArray);
+							console.log(
+								'buffFormArray',
+								Buffer.from(formattedArray.buffer)
+							);
+							console.log(
+								'buffFormArray',
+								formattedArray.toString()
+							);
+							console.log('ethereumAddress', ethereumAddress);
+							let input = '';
+							try {
+								input = await generate_input.generate_inputs(
+									Buffer.from(formattedArray.buffer),
+									ethereumAddress
+								);
+							} catch (e) {
+								console.log('Error generating input', e);
+								setDisplayMessage('Prove');
+								setStatus('error-bad-input');
+								return;
+							}
+							console.log(
+								'Generated input:',
+								JSON.stringify(input)
+							);
+
+							// Insert input structuring code here
+							// const input = buildInput(pubkey, msghash, sig);
+							// console.log(JSON.stringify(input, (k, v) => (typeof v == "bigint" ? v.toString() : v), 2));
+
+							console.time('zk-dl');
+							recordTimeForActivity('startedDownloading');
+							setDisplayMessage(
+								'Downloading compressed proving files... (this may take a few minutes)'
+							);
+							setStatus('downloading-proof-files');
+							await downloadProofFiles(filename, () => {
+								setDownloadProgress((p) => p + 1);
+							});
+							console.timeEnd('zk-dl');
```

Comment:
more linting issues -- these lines seemed way too tabbed forwards. Fixing tab size to be 2 spaces would be great.